_projects/2024/100546612/100546612.Rmd

```diff
@@ -0,0 +1,1007 @@
+---
+title: "Behind the numbers: comparing extreme poverty and GDP per capita by world regions"
+description: |
+  By recreating from group up a OurWorldinData chart about extreme poverty vs GDP per capita we look at the foundations of Data Visualisation techniques layer by layer. Further, this project also proposes X alternative representation for the original data to better understand the story behind the numbers. 
+categories: "2024"
+author: Mafalda González González
```

Comment:
González-González please; otherwise, the template interprets the first González as middle name.